content/nginx-one/agent/metrics/configure-otel-metrics.md

```diff
@@ -39,3 +39,195 @@ You can validate that metrics are successfully exported by using the methods bel
+    endpoint: "127.0.0.1:5643"
+    resource_to_telemetry_conversion:
+      enabled: true
+      namespace: test-space
+
+service:
+  pipelines:
+    metrics/prometheus-example-pipeline:
+      receivers:
+        - nginxplus
+      processors:
+        - resource/default
+      exporters:
+        - prometheus
+```
+
+{{% /tab %}}
+{{% tab name="NGINX OSS" %}}
+
+```yaml
+exporters:
+  prometheus:
+    endpoint: "127.0.0.1:5643"
+    resource_to_telemetry_conversion:
+      enabled: true
+      namespace: test-space
+
+service:
+  pipelines:
+    metrics/prometheus-example-pipeline:
+      receivers:
+        - nginx
+      processors:
+        - resource/default
+      exporters:
+        - prometheus
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+- **Third-party OTel Collector**
+
+{{< tabs name="third-party-collector" >}}
+{{% tab name="NGINX Plus" %}}
+
+```yaml
+exporters:
+  otlp/local-collector:
+    endpoint: "my-local-collector.com:443"
+    timeout: 10s
+    retry_on_failure:
+      enabled: true
+      initial_interval: 10s
+      max_interval: 60s
+      max_elapsed_time: 10m
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    metrics/otlp-example-pipeline:
+      receivers:
+        - nginxplus
+      processors:
+        - resource/default
+      exporters:
+        - otpl/local-collector
+```
+
+{{% /tab %}}
+{{% tab name="NGINX OSS" %}}
+
+```yaml
+exporters:
+  otlp/local-collector:
+    endpoint: "my-local-collector.com:443"
+    timeout: 10s
+    retry_on_failure:
+      enabled: true
+      initial_interval: 10s
+      max_interval: 60s
+      max_elapsed_time: 10m
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    metrics/otlp-example-pipeline:
+      receivers:
+        - nginx
+      processors:
+        - resource/default
+      exporters:
+        - otpl/local-collector
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+- **Add Debug Exporter**
+
+{{< tabs name="debug-exporter" >}}
+{{% tab name="NGINX Plus" %}}
+
+```yaml
+exporters:
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 200
+
+service:
+  pipelines:
+    metrics/debug-example-pipeline:
+      receivers:
+        - nginxplus
+      processors:
+        - resource/default
+      exporters:
+        - debug
+```
+
+{{% /tab %}}
+{{% tab name="NGINX OSS" %}}
+
+```yaml
+exporters:
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 200
+
+service:
+  pipelines:
+    metrics/debug-example-pipeline:
+      receivers:
+        - nginx
+      processors:
+        - resource/default
+      exporters:
+        - debug
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+
+### Troubleshooting
+
+To view the merged OpenTelemetry configuration set the NGINX Agent log level to "debug" in `/etc/nginx-agent/nginx-agent.conf`, and restart NGINX Agent:
+
+1. Edit the configuration file `sudo vim /etc/nginx-agent/nginx-agent.conf`
+1. Change the log property
+   ```yaml
+   log:
+     level: debug 
```

Comment:
``````suggestion
To view the merged OpenTelemetry configuration, change the NGINX Agent log level to "debug" in `/etc/nginx-agent/nginx-agent.conf`:

```yaml
 log:
   level: debug 
```

Then restart NGINX Agent.
``````

Not being prescriptive about editor, removing the list since it's just 2 items.